java-checks/src/test/java/org/sonar/java/checks/naming/BadMethodNameCheckTest.java

```diff
@@ -21,23 +21,33 @@
 
 import org.junit.jupiter.api.Test;
 import org.sonar.java.checks.verifier.CheckVerifier;
+import static org.sonar.java.checks.verifier.TestUtils.mainCodeSourcesPath;
 
 class BadMethodNameCheckTest {
 
   @Test
   void test() {
     CheckVerifier.newVerifier()
-      .onFile("src/test/files/checks/naming/BadMethodName.java")
+      .onFile(mainCodeSourcesPath("checks/BadMethodName.java"))
       .withCheck(new BadMethodNameCheck())
       .verifyIssues();
   }
 
+  @Test
+  void testWithoutSemantic() {
+    CheckVerifier.newVerifier()
+      .onFile(mainCodeSourcesPath("checks/BadMethodName.java"))
+      .withCheck(new BadMethodNameCheck())
+      .withoutSemantic()
+      .verifyIssues();
+  }
+
   @Test
   void test2() {
```

Comment:
Can we rename this test to be more indicative of what they are testing? Or perhaps split the 2 tests in their own test case? I see one test raising issues and the other none. But what is the implication of one failing and not the other?

Reply:
Fixed. Changed the name to be more descriptive.